gatsby-node.js

```diff
@@ -706,6 +706,21 @@ const createSectionPage = ({ envCreatePage, node }) => {
   });
 };
 
+exports.onCreateWebpackConfig = ({ actions, loaders, stage }) => {
```

Comment:
I added this due to the Reach-router issue. I could also have used a single child in the react element but it was not getting fit into my use case. @randychilau I'm curious about your views on this. 
![Reach-Router](https://github.com/layer5io/layer5/assets/65453826/9f8e5059-8b5c-4051-a4fd-b88edcd69809)

Reply:
I've reverted this one, as the build was failing.